pyproject.toml

```diff
@@ -42,6 +42,7 @@ dependencies = [
     'mne >= 1.1',
     'pooch',
     'torch',
+    'onnxruntime',
```

Comment:
Is there a way we can make both optional dependencies, since the user only needs to install one?